### DISCUSSION TITLE
Support for hex color codes in the source comments.

### DISCUSSION THREAD

**Discussion by rishabkshk** [codefori/vscode-ibmi#963] [General]
Support for hex color codes in the source comments. 
VS Code View:
![image](https://user-images.githubusercontent.com/21074746/202650033-b7c0b55d-5a7f-4332-b7cc-34733db8684d.png)

SEU View:
![image](https://user-images.githubusercontent.com/21074746/202650504-2989ebd9-bda1-4241-b30f-e44eb20ff390.png)

I am not sure if there already a way to ignore or replace them. The main issue occurs when I save the source with these junk characters from vs code. It adds '?' or sometimes '\Z' in place of those hex color codes to the source.
![image](https://user-images.githubusercontent.com/21074746/202651208-0bf60e9a-6def-4816-992e-03f54931e8f8.png)

![image](https://user-images.githubusercontent.com/21074746/202651449-baa7c818-c6e0-4c7f-850c-9b336520b49e.png)

**Comment by worksofliam** [comment]
Basic source/SEU colours can be turned on in the VS Code settings!

**Reply by Siggy2k** [reply]
When Source Dates are enabled, the colors do not work

**Reply by rishabkshk** [reply]
That's why I was having issues. Thanks, but I guess this is a bug @worksofliam ?

**Reply by worksofliam** [reply]
There is no plans to support colours while source dates are enabled.
